torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -928,6 +952,29 @@ Operation listSetItem(const Node* node) {
   };
 }
 
+
+template<>
+Operation listSetItem<Shared<BoolList>, bool>(const Node* node) {
+  return [](Stack& stack) {
+    Shared<BoolList> list;
+    int64_t idx;
+    bool value;
+
+    pop(stack, list, idx, value);
+
+    int64_t list_size = list->elements().size();
+    auto normalized_idx = normalizeIndex(idx, list_size);
```

Comment:
why not just call `getItem` defined above?

Reply:
getItem returns the value at that index, this is setting the index. I can refactor the getting of the normalized index, though